src/Oscoin/API/HTTP/Handlers.hs

```diff
@@ -97,6 +97,27 @@ getBlock h = do
         Nothing ->
             respond notFound404 $ errBody "Block not found"
 
+lookupBlockByHeight
+    :: (Serialise (SealedBlock c tx s))
+    => Height
+    -> ApiAction c tx s i a
+lookupBlockByHeight h = do
+    result <- liftNode $ Node.lookupBlockByHeight h
+    case result of
+        Just blk ->
+            respond ok200 $ Ok blk
+        Nothing ->
+            respond notFound404 $ errBody "Block not found"
+
+lookupBlocksByHeight
+    :: (Serialise (SealedBlock c tx s))
+    => ApiAction c tx s i a
+lookupBlocksByHeight = do
+    start <- param' "start"
+    end   <- param' "end"
+    blocks <- liftNode $ Node.lookupBlocksByHeight (start, end)
```

Comment:
I think we might want to unwrap `OldestFirst` and just send a list over the wire. Because the encoding of `OldestFirst`, `NewestFirst` and the wrapped list are the same we don’t get anything out of this. We would also not need the serialise instance for the chrono constructors.

Reply:
d146be09